databases/week1/databases.sql

```diff
@@ -0,0 +1,233 @@
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (26, 'Empty Mr Fluffy\'s litterbox', NULL, '2017-09-28 03:09:06', '2017-10-13 10:38:34', '2017-12-20 23:37:18', 2, 8);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (27, 'Buy new dining room table and chairs', 'Ikea has some on sale', '2017-09-21 12:02:34', '2017-10-02 02:05:11', '2017-12-06 00:14:30', 1, 3);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (28, 'Renew buscard', '3 zones', '2017-10-07 22:47:51', '2017-10-09 15:50:03', '2017-12-01 14:25:40', 2, 6);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (29, 'Sign up for linkedin', 'Make the CV awesome! 😄', '2017-09-04 00:57:47', '2017-10-18 18:07:48', '2017-12-07 23:04:38', 3, 2);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (30, 'Remove facebook from phone', 'To avoid interruptions when working', '2017-10-26 17:15:07', '2017-10-13 03:36:47', '2017-12-19 11:10:02', 3, 4);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (31, 'Backup databases to external disk', 'Remember to store the disk in another physical location', '2017-09-09 17:32:33', '2017-10-01 21:18:59', '2017-12-23 14:21:01', 1, 2);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (32, 'Put up the new lamp in the hallway', NULL, '2017-10-15 05:45:54', '2017-10-16 14:05:35', '2017-12-29 02:29:26', 3, 3);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (33, 'Hang up paintings in living room', NULL, '2017-09-10 05:36:11', '2017-10-09 17:40:42', null, 3, 4);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (34, 'Buy plane ticket to Auckland', 'Check prices online first!', '2017-09-05 09:07:22', '2017-10-15 09:36:06', '2017-12-07 11:10:05', 1, 9);
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (35, 'Learn about NoSQL databases', 'MongoDB, CouchDB, etc.', '2017-10-20 01:41:53', '2017-10-04 07:19:56', '2017-12-23 10:13:42', 2, NULL);
+
+
+-- HOMEWORK
+
+-- Find out how many tasks are in the task table:
+
+SELECT * id FROM task;
+
+-- Find out how many tasks in the task table do not have a valid due date:
+
+SELECT * FROM task WHERE due_date='NULL';
+
+-- Find all the tasks that are marked as done
+
+SELECT * FROM task WHERE status_id='3';
+
+-- Find all the tasks that are not marked as done
+
+SELECT * FROM task WHERE status_id='1' OR status_id='2';
+
+-- Get all the tasks, sorted with the most recently created first
+
+SELECT * FROM task ORDER BY  created ASC;
+
+-- Get the single most recently created task
+
+SELECT * FROM task WHERE created ASC;
```

Comment:
Here you will get an error because your SQL is not correct, make sure to try it in workbench as well. If you write it correctly you get all the data again, what you have to do is to get only one row as result. From the this result( SELECT * FROM task ORDER BY  created ASC;) write the SQL try to get the single most recently created task.